src/components/summary-dokumentkrav/SummaryDokumentkrav.tsx

```diff
@@ -0,0 +1,44 @@
+import { BodyShort, Label } from "@navikt/ds-react";
+import api from "../../api.utils";
+import { DOKUMENTKRAV_SVAR_SEND_NAA } from "../../constants";
+import { IDokumentkrav } from "../../types/documentation.types";
+import { DokumentkravSvar } from "../dokumentkrav-svar/DokumentkravSvar";
+import { DokumentkravTitle } from "../dokumentkrav-title/DokumentkravTitle";
+
+interface IProps {
+  dokumentkrav: IDokumentkrav;
+}
+
+export function SummaryDokumentkrav({ dokumentkrav }: IProps) {
+  const hasUploadedFiles = dokumentkrav.svar === DOKUMENTKRAV_SVAR_SEND_NAA;
+
+  return (
+    <li>
+      {hasUploadedFiles && (
+        <Label as="p">
+          <a
```

Comment:
Hva skal lenken gjør, er det en preview eller lastned link? Jeg fikk bare blank siden når jeg teste det lokalt. :/

Reply:
Det er en lenke for å laste ned dokumenter, og da skal jeg se litt mer på det. :)

Reply:
Det er faktisk en lenke for å vise dokumenter i nytt vindu, ikke nedlasting som jeg først trodde. Den fungerer for meg, men jeg skal teste litt mer i dev for å forsikre meg om at lenken fungerer som tiltenkt.